src/utils/mocks.ts

```diff
@@ -0,0 +1,17 @@
+export const mockDataTable = {
+  '1': ['sub-718211', 'sub-718213', 'sub-718216'],
+  '2': ['28.4', '24.6', '43.6'],
+  '3': ['M', 'F', 'M'],
+};
+
+export const mockHeaders = {
+  '1': {
+    header: 'participant_id',
+  },
+  '2': {
+    header: 'age',
+  },
+  '3': {
+    header: 'sex',
+  },
+};
```

Comment:
this works, but given the pain we had with the old annotation tool where for every test we had to go through the UI to initialize state with a test file, I think it's worth the effort now to make it so we can just provide file or file object directly to the store, and have the store do the processing and restructuring into the data object itself.